fuzzers/nyx_libxml2_parallel/src/main.rs

```diff
@@ -1,38 +1,40 @@
 use std::path::{Path, PathBuf};
 
-use libafl::bolts::launcher::Launcher;
-use libafl::bolts::rands::StdRand;
-use libafl::bolts::shmem::ShMemProvider;
-use libafl::events::EventConfig;
-use libafl::Error;
 use libafl::{
-    bolts::{core_affinity::Cores, rands::RandomSeed, shmem::StdShMemProvider, tuples::tuple_list},
-    corpus::{Corpus, InMemoryCorpus, OnDiskCorpus, Testcase},
+    bolts::{
+        core_affinity::Cores,
+        launcher::Launcher,
+        rands::{RandomSeed, StdRand},
+        shmem::{ShMemProvider, StdShMemProvider},
+        tuples::tuple_list,
+    },
+    corpus::{Corpus, OnDiskCorpus, Testcase},
+    events::EventConfig,
     feedbacks::{CrashFeedback, MaxMapFeedback},
     inputs::BytesInput,
     monitors::MultiMonitor,
     mutators::{havoc_mutations, StdScheduledMutator},
     observers::StdMapObserver,
+    prelude::CachedOnDiskCorpus,
```

Comment:
This should come from `corpus`, prelude is jist an easier way to import everything at once

Reply:
fixed